src/sqip.js

```diff
@@ -0,0 +1,74 @@
+const fs = require('fs')
+const path = require('path')
+
+const SVGPlugin = require('./plugins/svg')
+const SVGOPlugin = require('./plugins/svgo')
+const PrimitivePlugin = require('./plugins/primitive')
+const Base64EncodePlugin = require('./plugins/base64')
```

Comment:
i wonder if we should at least export them as separate packages with something like https://github.com/lerna/lerna

Reply:
Good point, we are actually using it for `melody` at trivago.